.github/workflows/release.yml

```
@@ -0,0 +1,53 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          cache: 'npm'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)